bin/resources/GameIndex.yaml

```diff
@@ -34634,15 +34634,15 @@ SLPS-25070:
   name: "Tales of the Sunrise Heroes 2"
   region: "NTSC-J"
 SLPS-25071:
-  name: "Visual Mix - Ayumi Hamasaki Dome Tour 2001 [Disc1of2]"
+  name: "A Visual Mix - Ayumi Hamasaki Dome Tour 2001 A [Disc1of2]"
```

Comment:
What's with the extra A at the end? is that not at the beginning?

Reply:
Reverted.